CHANGES.md

```diff
@@ -10,14 +10,16 @@ release.
 
 ### Enhancements
 
-[#656](https://github.com/cylc/cylc-ui/pull/656) - Show runhead-limited tasks.
+[#656](https://github.com/cylc/cylc-ui/pull/656) - Show runahead-limited tasks.
 
 [#657](https://github.com/cylc/cylc-ui/pull/657) - Display a different icon, with
 a shadow underneath, for the Job component.
 
 [#658](https://github.com/cylc/cylc-ui/pull/658) - Allow user to set the order
 of cycle points.
 
+[#543](https://github.com/cylc/cylc-ui/pull/543) - Use deltas in GScan.
+
```

Comment:
This pull request solves several issues. After this gets merged, I will go through each issue updating it, and then create another PR with the changelog. I'm not sure if we won't have to drop some changes, so I didn't want to include the text here now and then maybe have to remove it later.